members/templates/members/family_details.html

```diff
@@ -1,215 +1,223 @@
-                    <td><p><a type="button" href="{% url 'person_update' person.id %}" class="btn btn-warning">Redigér</a></p></td>
-                  </tr>
-                {% endfor %}
-              </tbody>
-            </table>
-            {% endif %}
-          </section>
-          <br>
-          <a class="btn btn-primary" role="button" href="{% url 'person_add' 'CH' %}">Opret barn</a>
-          <a class="btn btn-primary" role="button" href="{% url 'person_add' 'PA' %}">Opret forælder</a>
-          <a class="btn btn-primary" role="button" href="{% url 'person_add' 'GU' %}">Opret værge</a>
-          <br>
-          <br>
-          {%if request_parents %}
-          <blockquote class="bg-warning">
-            <p>Opret mindst 1 forælder / værge med kontaktoplysninger, så vi i Coding Pirates nemt kan få fat i jer.</p>
-          </blockquote>
-          {%endif%}
-          {% if need_confirmation %}
-          <blockquote class="bg-danger">
-          <p> Det er længe siden du sidst bekræftede dine oplysninger. <a class="btn btn-success" href="{% url 'confirm_details' %}">Bekræft oplysninger</a>
-          </blockquote>
-          {%endif%}
-        </div>
-      </div>
+                <td><p>{{person.address}}</p></td>
+                <td><p>{{person.zipcode}} {{person.city}}</p></td>
+                <td><p>{{person.email}}</p></td>
+                <td><p>{{person.phone}}</p></td>
+                <td><p><a type="button" href="{% url 'person_update' person.id %}" class="btn btn-warning">Redigér</a></p></td>
+              </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+        {% endif %}
+      </section>
+      <br>
+      <a class="btn btn-primary" role="button" href="{% url 'person_add' 'CH' %}">Opret barn</a>
+      <a class="btn btn-primary" role="button" href="{% url 'person_add' 'PA' %}">Opret forælder</a>
+      <a class="btn btn-primary" role="button" href="{% url 'person_add' 'GU' %}">Opret værge</a>
+      <br>
+      <br>
+      {%if request_parents %}
+      <blockquote class="bg-warning">
+        <p>Opret mindst 1 forælder / værge med kontaktoplysninger, så vi i Coding Pirates nemt kan få fat i jer.</p>
+      </blockquote>
+      {%endif%}
+      {% if need_confirmation %}
+      <blockquote class="bg-danger">
+      <p> Det er længe siden du sidst bekræftede dine oplysninger. <a class="btn btn-success" href="{% url 'confirm_details' %}">Bekræft oplysninger</a>
+      </blockquote>
+      {%endif%}
+    </div>
+  </div>
+
+  {% if participating or waiting %}
+  <h2>Tilmeldte aktiviteter</h2>
+  <p>Dette er de aktiviteter jeres børn er tilmeldt. I kan klikke på navnet for hvert arrangement, for at se flere detaljer om hvor og hvornår det foregår. I kan også se deltagerlisten for arrangementet.</p>
+  <p>Hvis et arrangement har en rød <kbd>betal</kbd>-knap, er betalingen ikke registreret. Tryk på knappen for at betale.</p>
+  <div class="row">
+    <div class="col-12">
```

Comment:
Hvorfor lave rækker når de alligvel er fyldt til 12? 
Der er lidt for mange klasser der ikke gør det de skal.